spirv_glsl.cpp

```diff
@@ -10265,7 +10292,28 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 		break;
 
 	case OpTranspose:
-		GLSL_UFOP(transpose);
+		if (options.version < 120) // Matches GLSL 1.10 / ESSL 1.00
+		{
+			// transpose() is not available, so instead, flip need_transpose,
+			// which can later be turned into an emulated transpose op by
+			// convert_row_major_matrix(), if necessary.
+			uint32_t result_type = ops[0];
+			uint32_t result_id = ops[1];
+			uint32_t input = ops[2];
+
+			auto &input_e = get<SPIRExpression>(input);
```

Comment:
`get<SPIRExpression>` is always iffy, use `maybe_get` and assume that transpose state is false, if it doesn't exist.